src/gemm.jl

```diff
@@ -2,7 +2,14 @@
 ## Borrowed from Knet.jl, adapted for compile-time constants
 
 using LinearAlgebra
-using LinearAlgebra.BLAS: libblastrampoline, BlasInt, @blasfunc
+using LinearAlgebra.BLAS: BlasInt, @blasfunc
+
+if VERSION >= v"1.7-"
+    using LinearAlgebra.BLAS: libblastrampoline
+    libblas = libblastrampoline
```

Comment:
Does this need to be `const`? I don't recall if that helps inside of modules, but other aliases in NNib use it.

Reply:
Yeah the original implementation does use `const` - I dropped that because the linter becomes upset (`Invalid redefinition of constant.`) but could always put it back